src/main/java/com/google/sps/servlets/GenerateServlet.java

```diff
@@ -15,8 +15,12 @@
 package com.google.sps.servlets;
 
 import com.google.gson.Gson;
+import com.google.sps.data.Destination;
 import java.io.IOException;
+import java.util.*;
```

Comment:
I think this might be importing everything in java.util.  Is there a reason you added this?